vck/src/commonMain/kotlin/at/asitplus/wallet/fallbackcredential/FallbackCredentialScheme.kt

```diff
@@ -0,0 +1,17 @@
+package at.asitplus.wallet.fallbackcredential
+
+import at.asitplus.wallet.lib.data.ConstantIndex
+
+data class FallbackCredentialScheme(override val vcType: String? = null, override val isoDocType: String? = null, override val sdJwtType: String? = null) : ConstantIndex.CredentialScheme {
+    companion object : ConstantIndex.CredentialScheme {
+        override val schemaUri: String = "https://wallet.a-sit.at/schemas/1.0.0/unknown.json"
+    }
+
+    override val schemaUri: String = "https://wallet.a-sit.at/schemas/1.0.0/unknown.json"
+    override val isoNamespace: String? = null
```

Comment:
Didn't we want to add a "mdoc fallback" credential where the iso namespace is set (and also for SD-JWT?)
Because then, it doesn't support all representations.